libc/config/linux/aarch64/entrypoints.txt

```diff
@@ -552,17 +575,25 @@ if(LIBC_TYPES_HAS_FLOAT16)
     libc.src.math.canonicalizef16
     libc.src.math.ceilf16
     libc.src.math.copysignf16
+    libc.src.math.expf16
     libc.src.math.f16add
     libc.src.math.f16addf
+    libc.src.math.f16addl
     libc.src.math.f16div
     libc.src.math.f16divf
+    libc.src.math.f16divl
+    libc.src.math.f16fma
```

Comment:
I think `f16fma` was unnecessarily disabled (disabling `f16fmal` and `f16fmaf128` should have been enough), but all other disabled `float16` entrypoints except `expf16` caused internal errors in Clang 11, so unless we've bumped our minimum supported Clang version and updated Clang on the Buildbot workers, we can't enable these entrypoints yet.